pkg/cmd/server.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The MayaData Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+	"strings"
+
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	// defaultOpenebsNS default openebs namespace
+	defaultOpenebsNS string = "openebs"
```

Comment:
can we avoid hardcoded ones?
we should pick either from env variables.. if not provided, consider the namespace in which operator is running

Reply:
ok

Reply:
` namespace in which operator is running` this can be fetched from downward api through env variable. I'll following env variable: 
NAMESPACE - operator namespace
OPENEBS_NAMESPACE - openebs namespace
VELERO_NAMESPACE - velero namespace